src/incidencematrix.jl

```diff
@@ -88,3 +123,41 @@ function Base.resize!(M::IncidenceMatrix{Symmetric}, n::Base.Integer)
     n >= 0 || throw(DomainError(n, "can not resize to a negative length"))
     _resize!(M,Int64(n),Int64(n))
 end
+
+function Base.show(io::IO, ::MIME{Symbol("text/plain")}, M::IncidenceMatrix)
+    m,n = size(M)
+    a,b = displaysize(io)
+    s = min(a - 5, size(M, 1))
+    print(io, "$m×$n IncidenceMatrix\n")
+    for i in 1:s
+        t = min(div(b, 2 + ndigits(size(M, 2))) - 1, length(row(M, i)))
+        print(io, "[")
+        join(io, [c for c in row(M, i)][1:t], ", ")
+        if (length(row(M,i)) > t)
+            print(io, ", …")
+        end
+        print(io, "]\n")
+    end
+    if (m > s)
+        print(io, "⁝")
+    end
+end
```

Comment:
```suggestion
function Base.show(io::IO, tp::MIME{Symbol("text/plain")}, M::IncidenceMatrix) = Base.show(IOContext(io), tp, M)
```
can't we just use the other function here?